package chap15_exception;

public class _01_Exception {

	public static void main(String[] args) {
		
		int[] intArr = new int[3];
		
		int idx = 0;
		
		while(true) {
			System.out.println(intArr[idx++]);
		}
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		

	}

}
